tests/unit/test_cli.py

```diff
@@ -427,51 +425,8 @@ def __init__(self, outputFormat='text'):
             self.baseUrl = 'baseUrl'
             self.verbose = 'verbose'
 
-    class FakeObject(protocol.message.Message):
-        __metaclass__ = python_message.GeneratedProtocolMessageType
-
-        FILE = descriptor.FileDescriptor(__file__, "test", "")
-        DESCRIPTOR = descriptor.Descriptor(
-            "FakeObject",
-            "test.FakeObject",
-            filename=__file__,
-            file=FILE,
-            containing_type=None,
-            fields=[
-                descriptor.FieldDescriptor(
-                    name="name",
-                    full_name="test.FakeObject.name",
-                    index=0,
-                    number=1,
-                    type=descriptor.FieldDescriptor.TYPE_STRING,
-                    cpp_type=descriptor.FieldDescriptor.CPPTYPE_STRING,
-                    label=descriptor.FieldDescriptor.LABEL_REQUIRED,
-                    default_value="",
-                    message_type=None,
-                    enum_type=None,
-                    containing_type=None,
-                    is_extension=False,
-                    extension_scope=None
-                ),
-                descriptor.FieldDescriptor(
-                    name="id",
-                    full_name="test.FakeObject.id",
-                    index=1,
-                    number=2,
-                    type=descriptor.FieldDescriptor.TYPE_STRING,
-                    cpp_type=descriptor.FieldDescriptor.CPPTYPE_STRING,
-                    label=descriptor.FieldDescriptor.LABEL_REQUIRED,
-                    default_value="",
-                    message_type=None,
-                    enum_type=None,
-                    containing_type=None,
-                    is_extension=False,
-                    extension_scope=None
-                )
-            ], nested_types=[], enum_types=[], extensions=[])
-
     def makeFakeObject(self):
```

Comment:
Ah, yeah - that's the same as https://github.com/ljdursi/ga4gh-client/blob/master/tests/unit/test_cli.py#L430 , but with the stricter C++ implementation it actually needs to be a .proto file that gets compiled into a _pb2.py .